api/openapi-spec/swagger.json

```diff
@@ -14460,89 +14460,6 @@
       },
       "type": "object"
     },
```

Comment:
This is fixed in https://github.com/kubernetes/kubernetes/pull/116705 and you may wait for that change to be merged and rebase with the latest code later.

Reply:
rebased and updated, thanks.